optaplanner-docs/src/modules/ROOT/pages/repeated-planning/repeated-planning.adoc

```diff
@@ -364,39 +364,38 @@ Usually it reuses the same instances of the problem facts and problem fact colle
 * The clone must use different, cloned instances of the entities and entity collections.
 Changes to an original Solution entity’s variables must not affect its clone.
 
-[[problemFactChangeExample]]
-==== Cloud balancing `ProblemFactChange` example
+[[problemChangeExample]]
+==== Cloud balancing `ProblemChange` example
 
-Consider the following example of a `ProblemFactChange` implementation in the cloud balancing use case:
+Consider the following example of a `ProblemChange` implementation in the cloud balancing use case:
 
 [source,java,options="nowrap"]
 ----
     public void deleteComputer(final CloudComputer computer) {
-        solver.addProblemFactChange(scoreDirector -> {
-            CloudBalance cloudBalance = scoreDirector.getWorkingSolution();
-            CloudComputer workingComputer = scoreDirector.lookUpWorkingObject(computer);
+        solver.addProblemChange((cloudBalance, problemChangeDirector) -> {
+            CloudComputer workingComputer = problemChangeDirector.lookUpWorkingObject(computer);
+            if (workingComputer == null) {
+                // The computer has already been deleted (the UI asked to changed the same computer twice), so do nothing
+                return;
+            }
```

Comment:
Is it ok to always sweep this under the mat?
I'd not add this if condition, because users will copy-paste this code and it's often better that by default calling to delete something twice fails - so they can figure out why they're calling that twice (maybe they meant to delete something else).